src/charm.py

```diff
@@ -62,6 +62,11 @@ def __init__(self, *args):
 
     def _configure(self, event: EventBase) -> None:
         """Config changed event."""
+        if invalid_configs := self._get_invalid_configs():
```

Comment:
NAD definitions are created at line 39 under constructor. We also require to validate the configs before configure.

Reply:
In _get_invalid_configs, we also need to validate MTU sizes. We decided to allow the range 1200 - 64K.

Reply:
So what happens if the MTU is bad?  The validation happens after we attempt to use it.  Either way, you are right, I was first thinking of validating the MTU, but this check can go after the multus stuff is successful